src/api/hydrateCharge.ts

```diff
@@ -0,0 +1,41 @@
+import { PAY_HYDRATE_CHARGE } from '../network/definitions/pay';
+import { sendRequest } from '../network/request';
+import { UNCAUGHT_HYDRATE_CHARGE_ERROR_CODE } from '../pay/constants';
+import { getPayErrorCode } from '../pay/utils/getPayErrorCode';
+import type {
+  HydrateChargeAPIParams,
+  HydrateChargeParams,
+  HydrateChargeResponse,
+} from './types';
+
+export async function hydrateCharge({
+  sender,
+  chargeId,
+}: HydrateChargeParams): Promise<HydrateChargeResponse> {
+  try {
+    const res = await sendRequest<
+      HydrateChargeAPIParams,
+      HydrateChargeResponse
+    >(PAY_HYDRATE_CHARGE, [
+      {
+        sender: sender,
+        chainId: 8453,
+        chargeId,
+      },
+    ]);
+    if (res.error) {
+      return {
+        code: getPayErrorCode(res.error?.code),
```

Comment:
So code for OnchainKit, it's about the location of the error.
In this case it will be
`AmHCa01` which stands for Api module Hydrate Charge api O1 error

Reply:
Ok, I've updated error sites to have codes in this format.